presto-main/src/test/java/com/facebook/presto/sql/planner/TestLogicalPlanner.java

```diff
@@ -60,32 +60,20 @@ public TestLogicalPlanner()
     }
```

Comment:
I'm not sure I understand how these changes fix the issue described in the commit message. It seems like a minor change in the organization of the tests.